packages/contracts/contracts/IExecutorSelectionStrategy.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/**
+ * @notice Interface for the Executor Selection Strategy (ESS).
+ */
+interface IExecutorSelectionStrategy {
+    /**
+     * @notice Queries the selected executor for a given project/bundle.
+     *
+     * @param _project  Address of the ChugSplashManager that mananges the project.
+     * @param _bundleId ID of the bundle currently being executed.
+     *
+     * @return Address of the selected executor.
+     * @return Time when the executor was selected.
```

Comment:
Actually let's get rid of the time here. It adds unnecessary complexity. We should just pass the time since the bundle was approved.